src/main/java/com/acme/conferencesystem/cfp_proposals/http/ProposalController.java

```diff
@@ -44,4 +50,16 @@ public ResponseEntity<Proposal> getProposalById(@PathVariable UUID id) {
                 .orElse(ResponseEntity.notFound().build());
     }
 
+    @PatchMapping("/{id}/approve")
+    public ResponseEntity<Void> approveProposal(@PathVariable UUID id) {
+        proposalService.approveProposal(id);
+        return ResponseEntity.ok().build();
+    }
+
+    @PatchMapping("/{id}/reject")
+    public ResponseEntity<Void> rejectProposal(@PathVariable UUID id) {
+        proposalService.rejectProposal(id);
+        return ResponseEntity.ok().build();
```

Comment:
If Patch doesn't return content, The response status for no content should be 204 not 200

Reply:
But maybe will be more practical to return the resource itself, in this case OK http status will be fine.

Reply:
https://google.aip.dev/121

Reply:
but let's merge ;)

Reply:
Yes, I was also considering return the resource itself. I'll do the changes in a separate PR, I just wanted to finish soon and unblock you